templates/react/package.json

```
@@ -3,14 +3,17 @@
   "private": true,
   "scripts": {
     "build": "pnpm recursive run build",
-    "dev": "run-pty % pnpm dev:node % pnpm dev:client % pnpm dev:contracts",
+    "dev": "pnpm recursive run dev",
```

while you're here, can you add `initialize` before recursive `dev`?
```suggestion
    "dev": "pnpm run initialize && pnpm recursive run dev",
```